fdmi/fol_fdmi_src.c

```diff
@@ -70,6 +70,40 @@
  *   still implement the counter -- to put in a work-around for future
  *   expansion.)
  *
+ * @section FDMI FOL records pruning on plugin side
+ *
+ * FDMI FOL records may be persisted on FDMI plugin side. In this case FDMI
+ * plugin must handle duplicates (because the process with FDMI source may
+ * restart before receiving "the message had been received" confirmation from
+ * FDMI plugin). In case if the source of FDMI records is CAS there might also
+ * be duplicates (i.e. FDMI FOL record about the same KV operation), and they
```

Comment:
s/FDMI FOL record/FDMI FOL records

Reply:
The point is that it might be _another_ FDMI FOL record about the same KV operation. Clarified & updated.

Please let me know if it's still not clear.